README.md

```diff
@@ -1,19 +1,23 @@
 # jetstream-config
 
-Custom configs for experiments analyzed in [jetstream](https://github.com/mozilla/jetstream). 
+Custom configs for [experiments](#custom-experiment-configurations) and [outcome snippets](#outcome-snippets)
+analyzed in [jetstream](https://github.com/mozilla/jetstream).
 
 ## Adding Custom Configurations
 
 Custom configuration files are written in [TOML](https://toml.io/en/).
-The name of the configuration file must match the **Normandy slug** of the experiment it is targeting.
 
 To add or update a custom configuration, open a pull request. CI checks will verify your configuration.
 Once CI completes, you may merge the pull request, which will trigger Jetstream to re-run your analysis.
 
-## Example configuration and syntax
+### Custom experiment configurations
+
+The name of the custom experiment configuration files must match the **Normandy slug**
+of the experiment it is targeting.
 
 Configuration files have four main sections:
-`[experiment]`, `[metrics]`, `[data_sources]`, and `[segments]`.
+[`[experiment]`](#experiment-section), [`[metrics]`](#metrics-section), [`[data_sources]`](#defining-data-sources),
+and [`[segments]`](#defining-segments).
```

Comment:
I'm not sure about this one. Should outcome snippets allows to define segments? Or just metrics and data sources?

Reply:
Fascinating question. Let's say "no" for now; I can imagine some use cases but I don't think they're critical.